test/e2e/index.tsx

```diff
@@ -13,24 +13,23 @@ interface Fixture {
 const fixtures: Fixture[] = [];
 
 // @ts-ignore
-const requireFixtures = require.context('./fixtures', true, /\.(js|ts|tsx)$/);
-requireFixtures.keys().forEach((path: string) => {
-  // require.context contains paths for module alias imports and relative imports
-  if (!path.startsWith('.')) {
-    return;
-  }
+const requireFixtures = import.meta.glob('./fixtures/**/*', { eager: true });
```

Comment:
Maybe rename t something else than 'require'? Is the `// @ts-ignore` still necessary?

Reply:
To avoid ts-ignore we need `glob` to be available in types.
We can keep it for a follow-up done in https://github.com/mui/mui-x/pull/17182/files#diff-c0d527bf7b08c376a02d793f515e4ae3ae5cdda654c6e66a75f8a99927fd23a8R12. 🤷